keps/sig-cluster-lifecycle/kubeadm/20190506-kubeadm-machine-output.md

```diff
@@ -0,0 +1,360 @@
+}
+```
+
+### Risks and Mitigations
+
+**Risk**:
+
```

Comment:
"Not introducing versioned output" maybe a risk here with the mitigation that a firm decision is made as part of the graduation process.

Reply:
Hi @randomvariable,

Ah, good call. I'll add that and link to the more verbose note about the topic. Thanks!

Reply:
Fixed